README.md

```diff
@@ -54,6 +54,40 @@ foo/include
 foo/lib
 ```
 
+## Compilation and installation with Visual Studio on Windows
+
+### Generate the project solution
+
+#### Build jansson
+Get [jansson](https://github.com/akheron/jansson) source code and checkout the latest version. In root folder run
+```
+cmake -G "Visual Studio 14 2015" -H./ -B./build
+```
+Open `build/jansson.sln` and build the solution. Copy content of `build/include` and `build/lib/Release` and put them in a common folder. When using CMake, specify `-DJANSSON_ROOT` to specify the path to include files and lib.
+
+#### Get OpenSSL and Boost
+First download development Windows OpenSSL binaries for example from [slproweb](https://slproweb.com/products/Win32OpenSSL.html) and run the installer. If you chose default installation path, add `C:\OpenSSL-Win32\bin` to your system `Path`.
+
+For running tests, you will also need `applink.c` from OpenSSL project which is required when compiling a `/MD DLL` on Windows.
+Get the file from [openssl](https://github.com/openssl/openssl) `/ms` directory and put it into `C:\OpenSSL-Win32\include\openssl`. This step is necessary because slproweb does not include this file in their releases. Some other OpenSSL binaries provider might, so check whether the file exists or not.
+
+If you want to run unit tests you should also get Boost sources.
+
+Using CMake, generate the project with
+```
+cmake -G "Visual Studio 14 2015" -DJANSSON_ROOT=C:/path/to/jansson -H./ -B./build
+```
+if you want to include unit tests and Boost is in non-standard system path, tell CMake where to find it:
+```
+cmake -G "Visual Studio 14 2015" -H./ -B./build -DJANSSON_ROOT=C:/path/to/jansson -DBOOST_ROOT=C:/path/to/boost -DBoost_NO_BOOST_CMAKE=TRUE
```

Comment:
CMake is smart enough to figure out the include and library directory from BOOST_ROOT so unless you have a non-standard boost install it should work fine.